mxabierto/adela
# Fix changing emai by user name

Changelog:

Changin the email by username when the user forgot his password.

M: app/views/devise/mailer/reset_password_instructions.html.haml

Do the follow the next steps:

1) Downlaod this source and start up your local environment.

2) Modify in your local environment:

A) add the gem letter_opener_web like this:

group :development do
gem 'letter_opener_web'
end

B) In the file config/initializers/devise.rb modify the next:

Coment the next line:
#config.mailer_sender = ENV['MAILER_FROM']

Add the next two lines
ENV['MAILER_FROM'] = 'your-valid-email@domain.com'
config.mailer_sender = ENV['MAILER_FROM']

3) run bundler

How to test:

A) In the login page http://localhost:port/
B) Press the button " Olvidaste tu contraseña"
C) In the next page type a valid mail and press the button "Enviar"
D) The output you must see is a windows which simulate the mail.

The original Fix:

Hola "value of the email"!

Now you must see:

Hola "value of the user name"!
